examples/gaussian_sims_txpipe/gaussian_sims.yml

```diff
@@ -0,0 +1,90 @@
+tjpcov:
+  use_mpi: True
+
+  # False: harmonic space; True: configuration space
+  do_xi: False
+
+  # harmonic space sacc input
+  # cl_file: /global/projecta/projectdirs/lsst/groups/WL/users/jprat/gaussian_sims_srdnzs_fullsky/022422/12300area/nside128/twopoint_data_fourier.sacc
+  cl_file: /global/projecta/projectdirs/lsst/groups/WL/users/jprat/gaussian_sims_srdnzs_fullsky/022422/12300area/nside128/summary_statistics_fourier_tjp.sacc
+
+  # 'set' from parameters OR pass CCL cosmology object OR yaml file
+  cosmo: examples/gaussian_sims_txpipe/fiducial_cosmology.yml
+
+  # allowed covariance type: gauss, ng (to be implemented)
+  cov_type: gauss
+
+  # Setting mask OR fsky approximation
+  mask_file: 
+    {% for i in range(5) %} 
+    source_{{ i }}: /global/projecta/projectdirs/lsst/groups/WL/users/jprat/gaussian_sims_srdnzs_fullsky/022422/12300area/nside128/source_maps.hdf5
+    # lens_{{ i }}: /global/cscratch1/sd/carlosgg/cosmodc2_outdir_128/mask_wo_density_unseens.fits.gz
+    lens_{{ i }}: /global/projecta/projectdirs/lsst/groups/WL/users/jprat/gaussian_sims_srdnzs_fullsky/022422/12300area/nside128/mask_nside128.hdf5
+    {% endfor %}
+
+  mask_names:
+    {% for i in range(5) %} 
+    source_{{ i }}: lensing_weight_{{ i }}
+    lens_{{ i }}: mask
+    {% endfor %}
+
+  outdir: /global/cscratch1/sd/carlosgg/cosmodc2_outdir_128/ 
+
+  # linear bias for lenses constant for redshift bin (example notebook)
+  {% for i in range(5) %} 
+  bias_lens_{{ i }}: 1
+  {% endfor %}
+
+  # IA: 0.5
+
+  binning_info: ignore
+
+  # Needed to read the masks and to generate nell because the window functions
+  # in the sacc file seems to be wrong
+  nside: 128
+
+NaMaster:
+  module: pymaster
+  f:
+    n_iter: 0
+    lmax_sht: -1
+  w:
+    lmax_mask: -1
+    l_toeplitz: -1
+    l_exact: -1
+    dl_band: -1
+    n_iter: 3
+    recompute: True
+  cw:
+    l_toeplitz: -1
+    l_exact: -1
+    dl_band: -1
+    n_iter: 3
+    recompute: True
+
+cache:
+  workspaces: 
+   '00': 
```

Comment:
Is there some documentation on how to use the 'cache' option in the input?

Reply:
I've added a short comment in that file. Basically, for now, I would point the users to have a look the documentation of the functions that use it.